plugin/src/main/java/hudson/plugins/findbugs/FindBugsPublisher.java

```diff
@@ -180,19 +221,32 @@ public String getIncludePattern() {
         return includePattern;
     }
 
+    /**
+     * @see {@link #getIncludePattern()}
+     */
+    @DataBoundSetter
+    public void setIncludePattern(String includePattern) {
+        this.includePattern = includePattern;
+    }
+
     @Override
     public Action getProjectAction(final AbstractProject<?, ?> project) {
         return new FindBugsProjectAction(project);
     }
 
     @Override
-    public BuildResult perform(final AbstractBuild<?, ?> build, final PluginLogger logger) throws InterruptedException, IOException {
+    public BuildResult perform(final Run<?, ?> build, final FilePath workspace, final PluginLogger logger) throws InterruptedException, IOException {
         logger.log("Collecting findbugs analysis files...");
 
-        String defaultPattern = isMavenBuild(build) ? MAVEN_DEFAULT_PATTERN : ANT_DEFAULT_PATTERN;
+        String defaultPattern = ANT_DEFAULT_PATTERN;
+        boolean isMavenBuild = build instanceof AbstractBuild && isMavenBuild((AbstractBuild) build);
```

Comment:
It's like moving the ```instanceof``` inside ```isMavenBuild``` and as a reward we have another backward compatibility issue (since ```isMavenBuild``` is protected in the superclass so there could be more plugins out there overriding it).